src/main.cpp

```diff
@@ -274,12 +252,10 @@ int main(int argc, char **argv) {
       console::EchoInfo("Bye");
       break;
     }
-    if (query->empty())
-      continue;
+    if (query->empty()) continue;
     try {
       auto ret = query::ExecuteQuery(session.get(), *query);
-      if (ret.records.size() > 0)
-        Output(ret.header, ret.records, output_opts, csv_opts);
+      if (ret.records.size() > 0) Output(ret.header, ret.records, output_opts, csv_opts);
```

Comment:
I would fix this on every occurrence of format changes
```suggestion
      if (ret.records.size() > 0) {
        Output(ret.header, ret.records, output_opts, csv_opts);
      }
```

Reply:
Fixed in clang PR: #37